src/routers/daoRouter.jsx

```diff
@@ -122,6 +123,9 @@ const DaoRouter = () => {
             isMember={isMember}
           />
         </Route>
+        <Route exact path={`${path}/gallery/minion/:minion`}>
```

Comment:
i was under the impression this would be a universal route. 'view gallery' on each minion vault links to the aggregate gallery or is there a separate gallery view for each minion also? we could pass in a minion for default filter, if needed?